media-gfx/openvdb/openvdb-7.1.0.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
```

Comment:
Same comments for this as the others.

Reply:
Openvdb builds fine with the default ninja backend. I no longer think this is needed and have removed it. Originally I thought it helped to prevent build failures when I was testing on a VM.